lib/appliance_console/external_httpd_configuration.rb

```diff
@@ -18,8 +19,8 @@ module ExternalHttpdConfiguration
       HTTP_KEYTAB         = "/etc/http.keytab"
 
       EXTERNAL_AUTH_FILE  = "conf.d/cfme-external-auth"
-      HTTPD_EXTERNAL_AUTH = "/etc/httpd/#{EXTERNAL_AUTH_FILE}"
-      HTTPD_CONFIG        = "/etc/httpd/conf.d/cfme-https-application.conf"
+      HTTPD_EXTERNAL_AUTH = Pathname.new(MiqApache.root_dir).join("etc/httpd/#{EXTERNAL_AUTH_FILE}")
+      HTTPD_CONFIG        = Pathname.new(MiqApache.root_dir).join("etc/httpd/conf.d/cfme-https-application.conf")
```

Comment:
If  MiqApache.config_dir returns a pathname then it's just
MiqApache.config_dir.join(...)
 On Feb 10, 2015 6:44 PM, "Brandon Dunne" <notifications@github.com> wrote:

> In lib/appliance_console/external_httpd_configuration.rb
> <https://github.com/ManageIQ/manageiq/pull/1639#discussion_r24462659>:
>
> > @@ -18,8 +19,8 @@ module ExternalHttpdConfiguration
> >        HTTP_KEYTAB         = "/etc/http.keytab"
> >
> >        EXTERNAL_AUTH_FILE  = "conf.d/cfme-external-auth"
> > -      HTTPD_EXTERNAL_AUTH = "/etc/httpd/#{EXTERNAL_AUTH_FILE}"
> > -      HTTPD_CONFIG        = "/etc/httpd/conf.d/cfme-https-application.conf"
> > +      HTTPD_EXTERNAL_AUTH = Pathname.new(MiqApache.root_dir).join("etc/httpd/#{EXTERNAL_AUTH_FILE}")
> > +      HTTPD_CONFIG        = Pathname.new(MiqApache.root_dir).join("etc/httpd/conf.d/cfme-https-application.conf")
>
> I'd prefer to see this say
>
> HTTPD_CONFIG        = Pathname.new(MiqApache.config_dir).join("cfme-https-application.conf")
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/1639/files#r24462659>.
>